lib/coins/ada.js

```diff
@@ -0,0 +1,50 @@
+const BASE58 = '123456789ABCDEFGHJKLMNPQRSTUVWXYZabcdefghijkmnopqrstuvwxyz'
+const b58 = require('base-x')(BASE58)
+const cbor = require('cbor-sync')
+const CRC32 = require('crc-32'); 
+
+module.exports = {depositUrl, parseUrl}
```

Comment:
Please move this at the bottom of file

Reply:
They use this in all other coins files, so I pick this as a convention, so do you think is a good idea to move that?

Reply:
Ok then leave it at the top